src/core/connection.c

```diff
@@ -1844,6 +1844,12 @@ QuicConnStart(
     UdpConfig.OwningProcess = Configuration->OwningProcess;
 #endif
 
+    if (!Connection->Settings.IsSet.QTIPEnabled) {
+        UdpConfig.UseQTIP = FALSE;
```

Comment:
`UdpConfig` is defaulted to `{0}` above, so there is no need to set `FALSE`.

Reply:
Something weird seems to be happening on OpenSSL. Without this guard, weird failures start happening on the CI.